.env

```diff
@@ -0,0 +1,30 @@
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=2e222659c5b006df8d0e68cc33913706
+###< symfony/framework-bundle ###
+
+DB_USER=root
+DB_PASSWORD=secret
+DB_PORT=27017
+DB_VERSION=6.0
+DB_URL=mongodb://${DB_USER}:${DB_PASSWORD}@database:${DB_PORT}
+
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+# MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
+EMAIL_QUEUE_NAME=send-email
+AWS_SQS_VERSION=latest
+AWS_SQS_REGION=us-east-1
+AWS_SQS_ENDPOINT_BASE=localstack
+AWS_SQS_KEY=fake
+AWS_SQS_SECRET=fake
+LOCALSTACK_PORT=4566
+MESSENGER_TRANSPORT_DSN=sqs://localstack:$LOCALSTACK_PORT/000000000000/$EMAIL_QUEUE_NAME?sslmode=disable
+STRUCTURIZR_PORT=8080
+
+CADDY_MERCURE_JWT_SECRET=!ChangeThisMercureHubJWTSecretKey!
+SERVER_NAME=localhost
+
+MONGODB_URL=mongodb://database:27017
+MONGODB_DB=symfony
```

Comment:
It will be better to name the database like application

Reply:
Sorry its never used I should delete it